src/styles/index.scss

```diff
@@ -1,3 +1,14 @@
-body {
+@import './utils/variables';
+@import './blocks/calendar';
+
+* {
+  box-sizing: border-box;
   margin: 0;
 }
+
+.sections {
```

Comment:
The class name `.sections` is not very descriptive and doesn't follow the BEM methodology. Consider renaming it to something more specific or use BEM naming conventions. For example, if this class is a block, it should have a name that reflects its purpose or the content it wraps. If it's an element, it should be named according to its block and its own role. If it's a modifier, it should reflect the variation it applies to a block or an element.